relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -1404,8 +1410,8 @@ mod tests {
                     "os.name": "Windows",
                     "platform": "javascript",
                     "release": "1.2.3",
-                    "transaction": "GET /api/:version/users/",
-                    "transaction.op": "myop",
+                    "transaction": "gEt /api/:version/users/",
+                    "transaction.op": "mYOp",
```

Comment:
Note that this test contains mixed metrics from transactions and spans. This PR only changes span metrics.